src/iam.ts

```diff
@@ -75,19 +66,22 @@ export interface SetPolicyCallback {
   (err?: Error | null, acl?: Policy, apiResponse?: object): void;
 }
 
-/**
- * @typedef {object} Policy
```

Comment:
curious why we're dropping these jsdoc strings, are they not used anywhere I'm guessing?

Reply:
Not dropped, just moved downwards :) `tsc` rips the jsdoc comments above interfaces so it doesn't go into the compiled docs.

Reply:
👍 awesome.